roles/ovirt-vm-infra/library/ovirt_vms.py

```diff
@@ -0,0 +1,1255 @@
+#!/usr/bin/python
```

Comment:
Can you please add file similar to this:

 https://github.com/oVirt/ovirt-ansible/blob/master/roles/ovirt-cluster-upgrade/library/README.md

Explaning why this file is needed and when it can be removed.